events/dlc/ep1/ep1_flavor_events.txt

```diff
@@ -2747,6 +2831,29 @@ ep1_flavor.2000 = {
 			trigger = { scope:maa_type_1 = flag:levies }
 			text = ep1_flavor.2000.levies
 		}
+		
```

Comment:
`ep1_flavor.2000`, in `immediate` block there.s a section that analyzes your army: which MaAs you have.
Check every use of `number_maa_regiments_of_base_type` in this file. There should be mod MaA checks nearby.